<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="map_image">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>681</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="coords_pole">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="spn_pole">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
